LayoutTests/imported/w3c/web-platform-tests/shadow-dom/declarative/declarative-shadow-dom-opt-in-expected.txt

```diff
@@ -106,7 +106,7 @@ PASS innerHTML on a <wbr>
 PASS innerHTML on template
 PASS innerHTML on template, with nested template content
 PASS innerHTML on shadowRoot
-PASS DOMParser
+FAIL DOMParser assert_true: Shadow root NOT FOUND. expected true got false
```

Comment:
FWIW, this was a bogus test as well. I'll fix the test before landing, but I'll await the bots and review first.